client/lib/products-values/translations.js

```diff
@@ -317,37 +317,38 @@ export const getJetpackProducts = () => {
 			monthly: [ CONSTANTS.PRODUCT_JETPACK_SEARCH_MONTHLY ],
 		},
 		optionShortNamesCallback: ( productObject ) => {
-			const numberOfDefinedTiers = 5;
-			switch ( productObject.price_tier_slug ) {
-				case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_100_RECORDS:
-					return translate( 'Pricing Tier 1: Up to 100 records' );
-				case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_1K_RECORDS:
```

Comment:
This seems like a good cleanup.  Can we remove all the `JETPACK_SEARCH_TIER...` constants also, though?  I think they're unused after this.